packages/typography/src/Typography.story.tsx

```diff
@@ -25,6 +25,12 @@ storiesOf('Typography', module).add('Default', () => (
       <InlineCode className={displayBlock}>InlineCode</InlineCode>
       <Disclaimer className={displayBlock}>Disclaimer</Disclaimer>
       <Overline>Overline</Overline>
+      <Body>
+        Emma and Brooke love <Link>corgis</Link>.
```

Comment:
🐶

Reply:
Omg was testing this with Rob and forgot to remove, thank you

Reply:
👍